### PR TITLE
Adding documentation for Fetch Metadata, Cross-Origin Opener Poliy & Cross-Origin Embedder Policy

### PR DESCRIPTION
**Pull request by salcho**
Hello Struts devs!

We're really proud to have contributed to Struts by adding Fetch Metadata (https://github.com/apache/struts/pull/426) and COOP/COEP support (https://github.com/apache/struts/pull/432) in the past few months while CSP is still being reviewed (https://github.com/apache/struts/pull/430). This PR updates documentation to reflect the new interceptors (not CSP!), their parameters, usage and brief explanations of what these security mitigations are and how they work. 

We hope this will motivate Struts developers to enable brand new security mitigations on their way forward :)

### CODE REVIEW

File: source/core-developers/coep-interceptor.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Parameters
+
+- `exemptedPaths` - Set of opt out endpoints that are meant to serve cross-site traffic. Paths should contain leading slashes and must be relative. This field is empty by default.
+- `enforcingMode` - Boolean variable allowing the user to let COEP operate in `enforcing`, which blocks both resource and reports violations, or `report-only` mode, which only reports violations. Default value for field is `false`.
+- `disabled` - Boolean variable disabling and enabling COEP. Default value for field is `false`.
+
+## Examples
+
+```xml
+<interceptor name="coep" class="org.apache.struts2.interceptor.CoepInterceptor"/>
```
Comment (yasserzamani) [inline source/core-developers/coep-interceptor.md:32]: As you've already defined and added it to `struts-default.xml`.`defaultStack` at [here](https://github.com/apache/struts/pull/432/files#diff-710b29900cea21e85893cae43dd08c92) , this duplicate definition is not needed as far as I can remember but please wait if @apache/struts-committers acknowledge as well.

File: source/core-developers/coep-interceptor.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+<action  name="someAction" class="com.examples.SomeAction">
+    <interceptor-ref name="defaultStack">
```
Comment (yasserzamani) [inline source/core-developers/coep-interceptor.md:35]: Same here. I think it should be something like:
```xml
<action  name="someAction" class="com.examples.SomeAction">
	<interceptor-ref name="defaultStack">
		<param name="coepInterceptor.exemptedPaths">...
.
.
.
	</interceptor-ref>
```

File: source/core-developers/coop-interceptor.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- `mode` - The policy mode COOP should follow. Available modes are `same-origin`, `same-origin-allow-popups`, `unsafe-none`. Default mode is `same-origin`.
+
+## Examples
+
+```xml
+<interceptor name="coop" class="org.apache.struts2.interceptor.CoopInterceptor"/>
+
+<action  name="someAction" class="com.examples.SomeAction">
+    <interceptor-ref name="defaultStack">
+    <interceptor-ref name="coop">
```
Comment (yasserzamani) [inline source/core-developers/coop-interceptor.md:39]: (similar here)

File: source/core-developers/fetch-metadata-interceptor.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+- `exemptedPaths` - Set of opt out endpoints that are meant to serve cross-site traffic. Paths should contain leading slashes and must be relative. This field is empty by default.
+
+## Examples
+
+```xml
+<interceptor name="fetchMetadata" class="org.apache.struts2.interceptor.FetchMetadataInterceptor"/>
+
+<action  name="someAction" class="com.examples.SomeAction">
+    <interceptor-ref name="defaultStack">
+    <interceptor-ref name="fetchMetadata">
```
Comment (yasserzamani) [inline source/core-developers/fetch-metadata-interceptor.md:37]: (similar here)

**Review by yasserzamani — CHANGES_REQUESTED** [review]
wow! astonish work! :clap:  Absolutely contribution motivates us and Struts needs especially regarding security improvements, than you very much! :100: :1st_place_medal: 

Below a few nitpick but please also wait for others if I'm right with them.

**Comment by gchatz22** [comment]
Thank you for your comments @yasserzamani ! Applied your changes and as said will also be waiting for @apache/struts-committers opinion.

**Review by yasserzamani — APPROVED** [review]

**Comment by lukaszlenart** [comment]
LGTM 👍

**Pull request merged**